src/components/Tag/Tag.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import './Tag.css'
+
+
+/* Note: Call tag in the following format
+    <Tag tagType={'price'} text = {'$20'}/>
+    <Tag tagType={'cuisine'} text ={'mexican'}/>
+*/
+
+function Tag({tagType, text}) {
+    var temp = tagType.concat('Tag');
```

Comment:
generally we dont want use this as it may or may not cause us some weird problems in the future. 

Instead, you can do like a f-string in javascript: 
className=`{tagType}Tag`